frontends/java/soot/src/main/java/ossf/fuzz/introspector/soot/CallGraphGenerator.java

```diff
@@ -303,6 +309,10 @@ private String extractCallTree(CallGraph cg, SootMethod method, Integer depth, I
 		StringBuilder callTree = new StringBuilder();
 		Iterator<Edge> outEdges = cg.edgesOutOf(method);
 
+		if (method.getName().endsWith("init>")) {
```

Comment:
Could we make this a bit more general? So it's easier to extend, e.g. have a list of candidates that we want to remove.

Reply:
OK.